src/main/java/org/spongepowered/api/entity/ai/task/AITaskTypes.java

```diff
@@ -33,37 +33,48 @@
 
 import java.util.function.Predicate;
 
+/**
+ * An enumeration of all possible {@link AITaskType}s available in vanilla
+ * minecraft.
+ */
 public final class AITaskTypes {
 
     // SORTFIELDS:ON
 
     /**
-     * {@link AITask} where {@link Creature}s avoid other {@link Agent}s based on a {@link Predicate}.
+     * {@link AITask} where {@link Creature}s avoid other {@link Agent}s based
+     * on a {@link Predicate}.
      */
-    public static final AITaskType AVOID_ENTITY = DummyObjectProvider.createFor(AITaskType.class, "AVOID_ENTITY");
+    public static final AITaskType AVOID_ENTITY = dummy("AVOID_ENTITY");
 
     /**
-     * {@link AITask} where {@link RideableHorse}s run around while {@link Humanoid}s attempt to tame them.
+     * {@link AITask} where {@link RideableHorse}s run around while {@link
+     * Humanoid}s attempt to tame them.
      */
-    public static final AITaskType RUN_AROUND_LIKE_CRAZY = DummyObjectProvider.createFor(AITaskType.class, "RUN_AROUND_LIKE_CRAZY");
+    public static final AITaskType RUN_AROUND_LIKE_CRAZY = dummy("RUN_AROUND_LIKE_CRAZY");
 
     /**
      * {@link AITask} where {@link Agent}s swim in liquids.
      */
-    public static final AITaskType SWIMMING = DummyObjectProvider.createFor(AITaskType.class, "SWIMMING");
+    public static final AITaskType SWIMMING = dummy("SWIMMING");
 
     /**
      * {@link AITask} where {@link Creature}s walk around.
      */
-    public static final AITaskType WANDER = DummyObjectProvider.createFor(AITaskType.class, "WANDER");
+    public static final AITaskType WANDER = dummy("WANDER");
 
     /**
      * {@link AITask} where {@link Agent}s will "watch" other {@link Entity}s.
      */
-    public static final AITaskType WATCH_CLOSEST = DummyObjectProvider.createFor(AITaskType.class, "WATCH_CLOSEST");
+    public static final AITaskType WATCH_CLOSEST = dummy("WATCH_CLOSEST");
 
     // SORTFIELDS:OFF
 
+    private static AITaskType dummy(String fieldName) {
+        return DummyObjectProvider.createFor(AITaskType.class, fieldName);
```

Comment:
My argument is that the code of defining a constant would be too long sometimes (exceeding 150 characters in a line), so that this would somewhat make code a little bit more beautiful.